web-frontend/vue.config.js

```diff
@@ -0,0 +1,30 @@
+const WasmPackPlugin = require("@wasm-tool/wasm-pack-plugin");
+const webpack = require("webpack");
+const path = require("path");
+
+module.exports = {
+  chainWebpack: config => {
+    // rust wasm bindgen https://github.com/rustwasm/wasm-bindgen
+    config
+      .plugin("wasm-pack")
+      .use(WasmPackPlugin)
+      .init(
+        Plugin =>
+          new Plugin({
+            crateDirectory: path.resolve(__dirname, "./wasm-wrapper")
```

Comment:
```suggestion
            crateDirectory: path.resolve(__dirname, "wasm-wrapper")
```
I think `path.resolve` already joins those.